src/Bugsnag/Client.php

```diff
@@ -33,6 +33,22 @@ public function __construct($apiKey)
     }
 
     /**
+     * Returns client config object or a single property
+     * if provided
+     *
+     * @param string|null $prop
+     * @return mixed
+     */
+    public function getConfig($prop = null)
```

Comment:
Sorry, I meant in the unit test to verify the behavior. Is a unit test even necessary given it's just testing `array_recursive_merge` essentially? I was just going by the unit test for the same behavior for the `Error` class.